### DISCUSSION TITLE
Nerf The Fucking Sliders

### DISCUSSION THREAD

**Discussion by DaRealGitHubDoriitos** [FunkinCrew/Funkin#1658] [Ideas]
I didnt realise the silders we're broken until i played Week 7 on NG, during the middle part of Stress there is Long Sliders and they brought me from about to Blue Ball to Destroying the fuck out of tank man.

#NerfTheFuckingSlidersNow

**Comment by ghost** [comment]
what sliders?

**Comment by Awfyboy** [comment]
> what sliders?

I think he is talking about the sustain notes

**Comment by DaRealGitHubDoriitos** [comment]
Yes those, altought in other rythm games they are called Sliders, (Im used to Osu!)

**Comment by steviegt6** [comment]
> Yes those, altought in other rythm games they are called Sliders, (Im used to Osu!)

Name doesn't really matter. Sliders is a good enough word to use as anyone capable of thinking can infer what you mean, lol.

**Comment by htv04** [comment]
A good way to make the sustain notes better without destroying them visually (for example, in some earlier versions of the game the sustain notes were visibly split) could be to add a short timer that's active during a sustain note that tells the game when to give the player health during the sustain note sequence.

**Comment by aflacc** [comment]
Today I learned thats what people mean by Sliders in FNF

**Comment by GOOBOZA** [comment]
yes and also make it so you lose more fucking mother fucker health when you miss a fucking mother fcuking note
